Dockerfile

```diff
@@ -32,6 +20,8 @@ RUN apt-get update \
     python3-distutils \
     && apt-get clean && rm -rf /var/lib/apt/lists/*
 
+RUN pip install --upgrade pip
+
```

Comment:
PYTHONUNBUFFERED here as well in the bottom of the file

Reply:
Updated!